roxctl/helm/derivelocalvalues/derivelocalvalues.go

```diff
@@ -276,6 +278,12 @@ func derivePublicLocalValuesForCentralServices(ctx context.Context, namespace st
 						`{.spec.template.spec.containers[?(@.name == "central")].env[?(@.name == "ROX_TELEMETRY_STORAGE_KEY_V1")].value}`, ""),
 				},
 			},
+			"declarativeConfig": map[string]interface{}{
+				"mounts": map[string]interface{}{
+					"configMaps": retrieveDeclarativeConfigConfigMaps(ctx, k8s, declarativeConfigMounts),
```

Comment:
Is this tested somewhere?

Reply:
Yea I checked and there currently are not tests for the `derive-local-values` command.
I've added the manual test steps in the PR description, and created [ROX-14460](https://issues.redhat.com/browse/ROX-14460)